template.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "themes-cdn",
+    "version": "0.0.0",
+    "description": "GCWeb theme",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/wet-boew/themes-cdn.git"
+    },
+    "author": "StdGit",
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/wet-boew/themes-cdn/issues"
```

Comment:
Est ce qu'on voudrait pointer vers les issues de GCWeb à la place?

ex: https://github.com/wet-boew/gcweb/issues

Reply:
@shawnthompson This one is for you

Reply:
@StdGit the issue tracker on theme-cdn repository is deactivated

Reply:
Done